cms/djangoapps/contentstore/views/course.py

```diff
@@ -103,6 +104,8 @@
 
 log = logging.getLogger(__name__)
 
+MOBILE_COURSE_AVAILABLE_UI = WaffleSwitch(STUDIO_WAFFLE_NAMESPACE, 'mobile_course_available_enable')
```

Comment:
Can we use a feature flag for this instead of a waffle switch? It will save us a query. Also we can use that same flag to ignore this setting here https://github.com/edx/edx-platform/blob/a69fce77dcae88c861de634162b8f639abdd1ef6/lms/djangoapps/courseware/access.py#L884

And remove the `IgnoreMobileAvailableFlagConfig` completely.